<launch>

  <!-- GDB functionality -->
  <arg name="debug" default="false" />
  <arg unless="$(arg debug)" name="launch_prefix" value="" />
  <arg     if="$(arg debug)" name="launch_prefix" value="gdb --ex run --args" />

  <!-- Verbose mode -->
  <arg name="verbose" default="false" />
  <arg unless="$(arg verbose)" name="command_args" value="" />
  <arg     if="$(arg verbose)" name="command_args" value="true" />
  
  <arg name="robot_name" default="pepper" />

  <!-- Launch pick and place -->
  <node name="moveit_simple_actions_pepper" launch-prefix="$(arg launch_prefix)"
    pkg="romeo_moveit_actions" type="romeo_moveit_actions" output="screen">
    <rosparam command="load" file="$(find pepper_moveit_config)/config/kinematics.yaml"/>
    <rosparam command="load" file="$(find moveit_simple_grasps)/config/$(arg robot_name)_grasp_data.yaml"/>
    <param name="robot_name"     value="$(arg robot_name)" />
    <param name="test_step"      value="0.0" />
    <param name="x_min"          value="0.0" />
    <param name="x_max"          value="0.0" />
    <param name="y_min"          value="0.0" />
    <param name="y_max"          value="0.0" />
    <param name="z_min"          value="0.0" />
    <param name="z_max"          value="0.0" />
    <param name="left_arm_name"  value="left_arm" />
    <param name="right_arm_name" value="right_arm" />
  </node>

</launch>
